docs/storage.md

```diff
@@ -422,8 +421,29 @@ Usually, you can use a NFS or SMB share. But if you don't have any of those, you
 That's it!
 
 :::tip
-Don't forget to rescan your SR after adding ISO files. Rescan is done automatically every 10 minutes otherwise.
-:::
+Don't forget to rescan your SR after adding, changing, or deleting ISO files. Rescan is done automatically every 10 minutes otherwise.:::
+
+```
+xe sr-rescan uuid=<UUID>
+```
+
+
+It's almost as easy from the CLI.  To create the ISO SR yourself:
+1. Create a directory on the local filesystem to storage your ISOs
+2. Copy/move ISOs to this new location
+3. Create the ISO SR using xe sr-create
+4. You can add or update ISOs later by placing them into the directory you created in step #1
```

Comment:
I don't think it's a good idea to use `#` here. Just 1 is enough

Reply:
Tomatos/tomatos -- but it's your soup :smile. (I've removed the hash.)